### ISSUE TITLE
Add Notifications API to send user an alert outside the page when alarm goes off

### ISSUE THREAD

**Issue by CompSciLauren** [CompSciLauren/web-clock-time-tracker#19]
**Is your feature request related to a problem? Please describe.**
If the user is not currently on the web clock time tracker website, they will miss the alert. The alert should pop up even if the user is not currently viewing the website.

**Describe the solution you'd like**
[Using the Notifications API](https://developer.mozilla.org/en-US/docs/Web/API/Notifications_API/Using_the_Notifications_API), set the alarm so that it will send an alert to the user that can be seen even if the user is not currently viewing the website.

**Additional context**
Will want to test to make sure it works:
- When viewing the Clock page
- When viewing a different page of the website
- When in the browser viewing a different website, with the web clock site still open in a tab
- When not currently looking at the browser at all, with the web clock site still open in a tab

**Comment by CompSciLauren** [comment]
Working on this one currently

**Comment by CompSciLauren** [comment]
Implemented in commit 9215b66645f63d5fcbd5393c376d09e106c1aee4. Was able to get it to work on mobile, does not work on desktop yet. It also does not work if you leave the Clock page and go to a different part of the website, but the other conditions listed under "Additional context" have been met on mobile. This is still an improvement though since it wasn't quite working correctly on desktop yet before this change. I logged issue #28 to track it.
